front/src/pods/student/student.component.tsx

```diff
@@ -1,7 +1,12 @@
 import React from 'react';
 import TextareaAutosize from '@material-ui/core/TextareaAutosize';
 import Typography from '@material-ui/core/Typography';
+
+import Checkbox from '@material-ui/core/Checkbox';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+
 import * as innerClasses from './student.styles';
+import { useAutoScroll } from '../../common/hooks/use-Auto-Scroll';
```

Comment:
The `common`  folder has an alias. That means you can write the import like:

`import { useAutoScroll } from 'common/hooks/use-Auto-Scroll';`